src/css/style.css

```diff
@@ -1,4 +1,91 @@
-/* Your custom styles go here! */
+:root {
+    --grey-lightest: #f7f9fa;
+    --blue-light: rgba(43, 63, 86, 0.81);
+    --green: #72cc18;
+    --green-light: rgba(114, 204, 24, 0.68);
+}
+
+html, body {
+    margin: 0;
+    height: 100%;
+    font-family: "Helvetica Neue", Sans-Serif, sans-serif;
+    background: var(--blue-light);
+    color: var(--grey-lightest);
+}
+
+body a {
+    color: orange;
+}
+
+body a:hover {
+    color: darkorange;
+}
+
+ul {
+    list-style-type: square;
+    list-style-position: inside;
+}
+
+#participants li {
+    border-bottom: solid 1px darkorange;
+    margin-bottom: 2px;
+    padding: 1px;
+    color: var(--grey-lightest);
+    border-radius: 3px;
+}
+
 #container {
     margin-top: 52px;
+    height: 100%;
+}
+
+#call {
+    display: flex;
+    flex-direction: row;
+    height: 100%;
+}
+
+#callFrame {
+    width: 70%;
+    height: 100%;
+    padding-right: 15px;
+}
+
+#presence {
+    padding-top: 10px;
+}
+
+#sidebar {
+    width: 40%;
+    padding-top: 5px;
+}
+
+#inviteURL {
+    font-size: 0.7em;
+}
+
+.hidden {
+    display: none !important;
```

Comment:
`!important` is usually a red flag that your selector just needs to be more specific. If you put this to the bottom of the file, it will be applied last to avoid overwriting the display value, or you may need to have a "stronger" selector to not get overwritten. (I'm assuming that was the issue if you've added !important)

Reply:
I did think about this! But I think for something like this, I really wanted a single all-ruling way to hide ANY element that would override everything else. Adding a bunch of individual more specific selectors seems like it would be a bit of an anti-pattern here when I really just want "one hidden class to _reliably_ rule them all" in this case. Do you think this is enough of an edge case to warrant it, or is it considered really _really_ bad practice?

Reply:
Ehhh I get it but I still would change it personally. In a codebase this small with no other component libraries to customize, you should be able to style it without `!important` because there shouldn't be that type of conflict already. People will have different opinions on how much to avoid `!important` but it's a code smell for me unless it's truly unavoidable.

Using an ID to apply initial styles puts you in a situation where those styles will be hard to override (because the ID is stronger than a class) so then you need something like `!important` -- it escalates quickly. And, having the `.hidden` class above `#entry` is a bit of a red flag too because any styles used to override should be below the other declarations. I just tested it and moving the `hidden` class to the bottom without the `!important` doesn't solve the issue because the initial styles are on the ID. 
A few options:
- leave as is (not my recommendation)
- update the `style.display` of the element in the JS instead of applying a new class, since a class isn't enough in this case
- add a class to the elements with an ID (e.g. `.entry`) and use the class as the selector for CSS instead of the ID. Then moving the `.hidden` styles to the bottom of the CSS file and removing `important` should be fine. The ID is nice for selecting the element in JS but it complicates the CSS a bit if you're planning on changing those styles.

Reply:
I've spent a lot of time having to figure out how to override or refactor `!important`s in legacy codebases so yes I am grumpy about them haha

Reply:
Thank you for educating me >.< My front-end-fu is better for it. I ended up going with option three: styling the relevant elements a a class instead of ID

Reply:
Cool! That looks good. Although... the important is still there! 😆 I just tested it without the important and it works the same either way so I think it can officially be removed. 😇

Reply:
🤦 It had been a long day. Removed now!